test/obb/server/handlers/test_request.clj

```diff
@@ -1,11 +1,11 @@
 (ns obb.server.handlers.test-request
   (:use clojure.test)
-  (:require [obb.server.core :as core]
+  (:require [obb.server.http-component :as http-component]
             [clojure.edn :as edn]
             [ring.mock.request :as mock]))
 
 (defn parsed-response
   "Gets the response for a HTTP request"
   [method path]
-  (let [response (core/app (mock/request method path))]
+  (let [response (http-component/app (mock/request method path))]
```

Comment:
Nitpicking again. Feel free to disregard. Looking at the usage of the namespace here, it looks like it would make more sense to call it something different (maybe just `http`?). The namespace does not include exclusively the component and this call sounded strange to me. What do you think?